project/Versions.scala

```diff
@@ -0,0 +1,5 @@
+object Versions {
```

Comment:
👍 This is one of those things I'd like to see in a sbt common package that every build depens on.